lib/RecipientGateway.ts

```diff
@@ -112,14 +116,46 @@ export class RecipientGateway {
    * ```
    * @param recipientId The Trolley recipient ID (e.g. R-xyzzy)
    */
-  async remove(recipientId: string) {
-    const endPoint = buildURL('recipients', recipientId);
+  async remove(recipientId: string | string[]) {
+    let endPoint = "";
+    let recipients: string[] = [];
+
+    if (Array.isArray(recipientId)) {
+      recipients = recipientId;
+      endPoint = buildURL('recipients');
+    } else {
+      endPoint = buildURL('recipients', recipientId);
+    }
```

Comment:
@zero-bites Just to be clear, this won't break existing usage of `remove(recipientId)` , but just extend the usage so now you can also call `remove(recipientIds[])` and delete multiple recipients, right?

Reply:
Right, it won't break existing usage 👍